fix: added missing CSS unit when positioning the window offset to `top`

emeraldsanto opened the pull request:
### Description

This PR simply adds the missing `px` unit to the calculation of how much to offset from `top`. Without this, certain browsers will simply interpret the value as `0` which makes the page scroll to the top.

emeraldsanto committed:
fix: added missing CSS unit (#1)

topolanekmartin commented:
On mobile iOS it does not attach top property on body element. This causes the scroll is not returned back to its previous position after scroll is enabled.

pbt committed:
fix: Target window.top.document.body; remove code for adjusting viewport changes (#2)

1. Change the target to document.body ➡️ window.top.document.body. (In non-iframe situations window.top resolves to window.) This is because in certain situations we might want to invoke the body scroll lock while inside of an iframe and want to make sure locking extends beyond the iframe.

2. Remove the code that adjusts for viewport changes. While debugging this library I found that this passage of code that attempts to adjust for viewport changes did not work (it was causing the top value to be unset, meaning scroll position was not preserved). I don't think that it ought to be the responsibility of this library to adjust for viewport changes; it should just be to apply a body scroll lock. If they need to, consumers can adjust by listening to changes in visualViewport or using the new CSS display viewport dimensions.
